sql/core/src/test/resources/sql-tests/inputs/string-functions.sql

```diff
@@ -38,3 +38,11 @@ SELECT substring('Spark SQL', 5, 1);
 SELECT substring('Spark SQL' from 5);
 SELECT substring('Spark SQL' from -3);
 SELECT substring('Spark SQL' from 5 for 1);
+
+-- trim/ltrim/rtrim
+SELECT trim('yxTomxx', 'xyz');
+SELECT trim('xxxbarxxx', 'x');
+SELECT ltrim('zzzytest', 'xyz');
```

Comment:
Nit: add some characters of 'xyz' at the end

Reply:
Done